lib/concurrent/atomic/atomic_markable_reference.rb

```diff
@@ -1,6 +1,5 @@
 module Concurrent
-  module Edge
-
+  module Atomic
```

Comment:
There is no `Atomic` module. We are non-idiomatic in the way we handle some subdirectories and namespaces. We are going to fix this in 2.0. For now please leave `atomic_markable_reference.rb` in the `lib/concurrent/atomic` directory but make the namespace `Concurrent:: AtomicMarkableReference`.